core/urls.py

```diff
@@ -20,6 +21,14 @@
         ResourceListByCategoryView.as_view(),
         name='resource_list_view'
         ),
+    path(
+        'About_Us/',views.about,
+        name='about'
+        ),
+    path(
+        'Contact_Us/',views.contact,
```

Comment:
Bad naming convention-
`Contact_Us/` should be converted to `contact-us/`

Reply:
resolve